source/templates.tex

```diff
@@ -5698,8 +5698,8 @@
 An instantiated template specialization can be either implicitly
 instantiated\iref{temp.inst} for a given argument list or be explicitly
 instantiated\iref{temp.explicit}.
-A specialization is a class, variable, function, or class member that is either
-instantiated or explicitly specialized\iref{temp.expl.spec}.
+A \defn{specialization} is a type, variable, function, or class member that is either
+instantiated from a template or specialized for by an explicit specialization\iref{temp.expl.spec}.
```

Comment:
How about the change to how we specify explicit specializations? I think that is sufficiently editorial, since we know that templates are what we explicitly specialize.

Reply:
I think "explicitly specialized" (verb form) is used in [temp.expl.spec] sufficiently often that we don't need to work around the term.
Maybe "that is either instantiated from a template (xref) or produced by an explicitly specialization of a template (xref)."